src/server.js

```diff
@@ -51,6 +54,13 @@ module.exports = function() {
 		return res.sendFile(theme);
 	});
 
+	app.get("/packages/:package/:filename", (req, res) => {
+		const packageName = req.params.package;
```

Comment:
With stored map of loaded packages, verify requested package against that map here.

Reply:
I'm just checking that it exists